extensions-contrib/opentsdb-emitter/src/test/java/org/apache/druid/emitter/opentsdb/OpentsdbEmitterConfigTest.java

```diff
@@ -39,10 +39,41 @@ public void setUp()
   @Test
   public void testSerDeserOpentsdbEmitterConfig() throws Exception
   {
-    OpentsdbEmitterConfig opentsdbEmitterConfig = new OpentsdbEmitterConfig("localhost", 9999, 2000, 2000, 200, 2000, 10000L, null);
+    OpentsdbEmitterConfig opentsdbEmitterConfig = new OpentsdbEmitterConfig("localhost", 9999, 2000, 2000, 200, 2000, 10000L, null, "druid");
     String opentsdbEmitterConfigString = mapper.writeValueAsString(opentsdbEmitterConfig);
     OpentsdbEmitterConfig expectedOpentsdbEmitterConfig = mapper.readerFor(OpentsdbEmitterConfig.class)
                                                                 .readValue(opentsdbEmitterConfigString);
     Assert.assertEquals(expectedOpentsdbEmitterConfig, opentsdbEmitterConfig);
   }
+
+  @Test
+  public void testSerDeserOpentsdbEmitterConfigWithSpaceNamespace() throws Exception
```

Comment:
nit: rename `testSerDeserOpentsdbEmitterConfigWithSpaceNamespace` to `testSerDeserOpentsdbEmitterConfigWithNamespaceContainingSpace`.

Reply:
Well spotted, didn't think much of test names when refactoring. And actually, I should bring all test-names in line with same naming convention, for clarity. In this file the tests were named like `<...>Namespace<...>`, and in `EventConverterTest.java` they're called `<...>Prefix<...>`, they should all be `<...>NamespacePrefix<...>`. I'll update them all. :)

Reply:
committed